entries/entry01.md

```diff
@@ -1,7 +1,29 @@
 # Entry 1
-##### X/X/XX
+##### 10/18/19
 
-Text
+### Prologue
+After a couple weeks of researching I came to the conclusion that I don’t know what to do for the Freedom Project. There isn’t any particular issue I’m currently having that can be solved with software. I thought about making something like a planner or an application that makes my desktop look better. However none of them really speaks to me, they are just too cliche. I did more research exploring the many possibilities.
+
+### Decision
+After a couple hours it finally came to me. I want to make a game. A game that has a vast world generation. The type of game that I have been playing since I was little. One of the tools I will be using would be Ruby. I see that some games might require many tools and so I decided that Ruby will be one of them.
+
+### Reflection
+Although I said that I would use Java in class but now that I think about it, it wouldn't be fun if that's all I use. I want to learn some new languages as well as something my classmates might not have mentioned during class. After thinking about it, I settled with Ruby.
+
+### Why Ruby?
+I know that SEP12 students last year learn about Ruby. Because there was a change of course the class this year is primarily focused on Java. This only makes me more curious as to what it is and makes me want to learn it as well. Considering how Ruby is so underrated I see just as much opportunity as to what it can do compared with Java, Python, and C++.
+
+### Inspiration
+From what I know about Ruby, it is not practical in making games but I still want to try. I was inspired by a book called *Developing Games with Ruby* as surf the web for how I can make a game with Ruby. The boy at the beginning of the book is someone that I can relate. He too wanted to make a world just like me. Although he was not able to do it, he still learned something. I hope that I can get something out of this project as well.
+
+### Skills
+Undoubtedly the skills that I acquired is **How to Google** and **How to Read**. There is really little information on how to make games with Ruby therefore I need to search for ways. I used the tactic of shortening my question to only keyworks just so there will be more accurate results. While reading the Wikipedia and documents I mainly use CTRL + F to find word that I’m looking for. It really did help out a lot.
+
+### Progress
+As of current, I would consider my progress to be at the starting line. I have defined what I want to do and how. I also did researches as to what tools I might want to use, starting with Ruby because it’s the most alienating. I hope to see good results in my next entry.
```

Comment:
Ruby's greatest strength is making web applications. Java's greatest strength is its ability to make _anything_. It's much more versatile than Ruby (and also better at making games).